hwi.spec

```diff
@@ -1,21 +1,29 @@
 # -*- mode: python -*-
 import platform
 import subprocess
+import os
 
 block_cipher = None
 
 binaries = []
+
 if platform.system() == 'Windows':
-    binaries = [("c:/python3/libusb-1.0.dll", ".")]
+    libusb_path = 'c:/python3/libusb-1.0.dll'
```

Comment:
nit: Use double quotes rather than single quotes

Reply:
I used single quotes because most of the file uses it. Should I change the other parts as well?

Reply:
Just the changes that you have made.